app/events/db/db_table_field_mappings.yaml

```diff
@@ -541,3 +541,15 @@ mappings:
                 city: location_city
                 state: location_state
                 country: location_country
+
+    dictionary_topics_relevance:
+        name: dictionary_topics_relevance
+        schema: iiq_schema
+        unique_key: work_platform_id, identifier
```

Comment:
check what is the unique constraint in our table, and accordingly add here

Reply:
this may be this is directly coming from vendor, in that case this seems to be valid unique constraint